docs/install/k8s-proxy.md

```diff
@@ -0,0 +1,37 @@
+# Install the service-broker-proxy-k8s
+
+## Prerequisites
+
+* `tiller` is installed and configured in the Kubernetes cluster.
```

Comment:
Maybe only service-catalog with a link to their installation guid which includes tiller and helm 
- https://github.com/kubernetes-incubator/service-catalog/blob/master/docs/install.md